lfx-mentorship/2022/02-Summer/project_ideas.md

```diff
@@ -29,4 +29,18 @@ Project maintainers and mentors, please submit the ideas below (under the Propos
 
 ## Proposed Project ideas
 
+### Volcano
 
+#### Official Website Docs Enhancement
+
+- Description: Official website docs has not been updated for a long time including technology docs, talks, best practice and so on, which bothers users and developers a lot.  
+- Recommended Skills: Kubernetes, golang, English, Chinese
```

Comment:
I  suggest that you add "Technical writing" to the list of recommended skills here as well.

[K8s SIG-Network APIs did a similar project this term](https://github.com/cncf/mentoring/pull/454/files), they also asked for writing samples from applicants.

Reply:
I should also add, thanks so much for suggesting a non-code project, this is great!

Reply:
Thanks for your advice, @nate-double-u . I'm OK about that.

Reply:
I've added the skill to the description. Please take a review. Thanks!